### ISSUE TITLE
BBOX parameter not recognized in newest QGIS Server

### ISSUE THREAD

**Issue by lucernae** [kartoza/geonode#302]
Seen in: QGIS Server LTR: 2.14.16

```
<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc">
    <ServiceException code="InvalidParameterValue">BBOX is empty</ServiceException>
</ServiceExceptionReport>
```

Even though BBOX is included in WMS GetMap parameter.

**Comment by Gustry** [comment]
Can you copy/paste the full URL?
Strange.


BTW: They started promoting the new 2.18 LTR version: https://twitter.com/qgis/status/885764408386912256

**Comment by lucernae** [comment]
```
http://geonode.dev/qgis-server/ogc/?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image/png&LAYERS=test_grid&HEIGHT=250&WIDTH=250&SRS=EPSG:4326&BBOX=-5.2820,96.9406,-5.54025,97.1250
```

But you need to do ```docker pull kartoza/qgis-server:LTR``` first.
It was seen in travis:
travis-ci.org/kartoza/geonode/

**Comment by lucernae** [comment]
@Gustry I've found the problem. Apparenty you put a wrong bbox format, the x is swapped. But QGIS raise the wrong message as no BBOX. Now I don't know why it works in the first place.

**Comment by lucernae** [comment]
Fixed in #301 by swapping the bbox.
